### PR TITLE
Enable: conditional_returns_on_newline

### PR DESCRIPTION
**Pull request by lukabratos**
### What?

Conditional statements should always return on the next line.

### Why?

It's easier to set the breakpoint and debug in Xcode.

----
CC @pusher/mobile

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
 included:
   - Sources
+excluded:
+  - Sources/MD5.swift
```
Comment (lukabratos) [inline .swiftlint.yml]: `MD5.swift` is a dependency that will be removed in the future version of the SDK so we're excluding this file for now.

File: Sources/PushNotifications.swift [hunk 2]
```diff
@@ -319,22 +329,27 @@ import Foundation
...
-        guard FeatureFlags.DeliveryTrackingEnabled else { return .ShouldProcess }
+        guard FeatureFlags.DeliveryTrackingEnabled else {
+            return .ShouldProcess
+        }
 
         #if os(iOS)
             let applicationState = UIApplication.shared.applicationState
-        #endif
-
-        #if os(iOS)
```
Comment (lukabratos) [inline Sources/PushNotifications.swift]: Cleanup double `#if os(iOS)`.

File: .swiftlint.yml (hunk 1 above)
Comment (luismfonseca) [reply .swiftlint.yml]: These comments are best placed in the actual file.

```
excluded:
  - Sources/MD5.swift # this dependency will be removed in the future version of the SDK
```

**Comment by lukabratos** [comment]
Thanks @luismfonseca, I've added the comment. 

🚢 🎉

**Pull request merged**